workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectLinker.cs

```diff
@@ -8,13 +8,13 @@ namespace Improbable.Gdk.Core
     public class EntityGameObjectLinker
     {
         private readonly World world;
-        private MutableView view;
+        private readonly ILogDispatcher logDispatcher;
         private readonly HashSet<Type> gameObjectComponentTypes = new HashSet<Type>();
 
-        public EntityGameObjectLinker(World world, MutableView view)
+        public EntityGameObjectLinker(World world)
         {
             this.world = world;
-            this.view = view;
+            logDispatcher = new ForwardingDispatcher();
```

Comment:
Doesn't this need a connection to work properly? Also - shouldn't this inherit the type of logger that its owning worker has?